### PR TITLE
NDS-79 Phase banner component

### PR DESCRIPTION
**Pull request by wa-rren-dev**
https://nicedigital.atlassian.net/browse/NDS-79

### CODE REVIEW

File: components/nds-phase-banner/README.md [hunk 1]
```diff
@@ -1,11 +1,97 @@
...
+- Type: `string`
+
+Any additional className will be merged on to the container.
+
+### SCSS
+
+If you're not using [React](#react), then import the SCSS directly into your application by:
+
+```scss
+@import "~@nice-digital/nds-alert/scss/alert";
```
Comment (ediblecode) [inline components/nds-phase-banner/README.md:79]: This should be phase banner scss import

File: components/nds-phase-banner/README.md [hunk 2]
```diff
@@ -1,11 +1,97 @@
...
+```
+
+### HTML
+
+If you're not using [React](#react), then include the [SCSS as above](#scss) and use the HTML:
+
+```html
+<div class="alert alert--info">
```
Comment (ediblecode) [inline components/nds-phase-banner/README.md:87]: Same as CSS comment above - wrong markup snippet

File: components/nds-phase-banner/README.md [hunk 3]
```diff
@@ -1,11 +1,97 @@
...
+import { PhaseBaner } from "@nice-digital/nds-phase-banner";
+
+<PhaseBanner alpha>
+	This is a new service - your <a href='#'>feedback</a> will help us to improve it.
+</PhaseBanner>;
 ```
-const ndsPhaseBanner = require('@nice-digital/nds-phase-banner');
 
-// TODO: DEMONSTRATE API
+#### Props
```
Comment (ediblecode) [inline components/nds-phase-banner/README.md:30]: Is it worth also saying that any other props are spread over the wrapping element?

Comment (ediblecode) [reply components/nds-phase-banner/README.md:30]: I think we do this elsewhere in readmes

File: components/nds-phase-banner/__tests__/nds-phase-banner.test.js [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+
+	it("should merge additional classes", () => {
+		const wrapper = shallow(
+			<PhaseBanner alpha className="mt--0s">
+				Some content
+			</PhaseBanner>
+		);
+		expect(wrapper.find("p").props()["className"]).toContain("mt--0");
+		expect(wrapper.find("p").props()["className"]).toContain("phase-banner");
+	});
```
Comment (ediblecode) [inline components/nds-phase-banner/__tests__/nds-phase-banner.test.js:26]: Can we add a test for spreading additional props over the wrapping element?

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: components/nds-phase-banner/README.md (hunk 2 above)
Comment (wa-rren-dev) [reply components/nds-phase-banner/README.md:87]: 🤦

**Review by ediblecode — APPROVED** [review]

**Pull request merged**